utils/testrunner/Reporter.js

```diff
@@ -207,7 +207,8 @@ class Reporter {
       console.log(`${prefix} ${colors.red(`[TIMEOUT ${test.timeout}ms]`)} ${test.fullName} (${formatLocation(test.location)})`);
       if (test.output) {
         console.log('  Output:');
-        console.log(padLines(test.output, 4));
+        for (const line of test.output)
```

Comment:
This will indent poorly if a line of output is multiline.

Reply:
I'd like to be consistent with requre('debug')

Reply:
previously I could do `test.output += JSON.stringify(obj, undefined, 2)`. Now I can't without breaking the indent.

Reply:
We were never putting anything from stdio there at the first place, let's see how it looks first